cluster/juju/layers/kubernetes-worker/actions/registry

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+# Example usage :
+#
+# Registry creation
+#
+# juju run-action kubernetes-worker/0 registry domain=myregistry.internal \
+# htpasswd="$(cat htpass)" ingress=true \
```

Comment:
Disclaimer: this is probably an unpopular opinion.

I think it's an extra step. I'm not entirely opposed, but it seems unnecessary as the config values are not base64 encoded when placed on disk or in config. I presume you're doing this to ensure data consistency, that juju appears to be doing already with non base64 encoded values.